### PR TITLE
Add hero-card compound

### PR DESCRIPTION
**Pull request by ruxiang05**
# Description

The hero-card is a component needed on the money website to call out seasonal products.

Screenshot 2020-06-25 at 10 30 45

Screenshot 2020-06-25 at 10 19 27

# Checklist

Pull request contains:

- [x] A new component
- [ ] Component maintenance: improvement / bug fix / etc
- [ ] Component library change: storybook / webpack / etc

Definition of done:

- [ ] Includes theme changes for both Uswitch and money
- [ ] Work has been tested in multiple browsers
- [x] PR description includes description of change
- [x] PR description includes screenshot of change
- [ ] If new component, designer has approved screenshot
- [ ] If the change will affect other teams, that team knows about this change
- [x] If introducing a new component behaviour, added a story to cover that case.
- [ ] If component change, version updated

### CODE REVIEW

File: src/compounds/hero-card/src/index.tsx [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+}
+
+const styles = (element?: string) =>
+  `compounds.heroCard.variants${element ? `.${element}` : ''}`
+
+const HeroCard: React.FC<Props> = ({ title, ctaText, children }) => {
+  return (
+    <div
+      sx={{
+        display: 'grid',
```
Comment (barneyfoxuk) [inline src/compounds/hero-card/src/index.tsx:20]: In general we don't use grid (it's only used for product table right now as it was to only way to achieve that) due to it's limited support on IE11.  In general I would use flexbox where possible over grid, but if there's a specific reason for using it that's cool?

Comment (barneyfoxuk) [reply src/compounds/hero-card/src/index.tsx:20]: Chatted on a call and this is a good use case for grid. Will be tested in IE11 to confirm everything works

Comment (jonathan-fielding) [inline src/compounds/hero-card/src/index.tsx:20]: has this been tested in IE11

File: src/elements/link-list/src/index.tsx [hunk 2]
```diff
@@ -24,25 +24,29 @@ export const LinkList: React.FC<ListLinkProps> = ({
...
-          display: 'flex',
-          alignItems: 'center',
-          variant: styles(variant, 'header')
-        }}
-      >
-        {icon}
-        <Styled.h3
-          as="h2"
+      {(title || icon) && (
+        <div
```
Comment (jonathan-fielding) [inline src/elements/link-list/src/index.tsx:28]: I know this isnt new but as this is the title of the link list shouldn't this be a <header> element

File: src/themes/money/theme.json [hunk 3]
```diff
@@ -818,6 +822,29 @@
...
       }
+    },
+
+    "heroCard": {
+      "variants": {
+        "background": "linear-gradient(161.42deg, #924A8B 22.95%, #AF4C83 77.05%)",
+        "box-shadow": "0px 9px 19px rgba(92, 36, 87, 0.2)",
+        "column-gap": 104,
+        "paddingX":["sm","xxxl"],
+        "pt": ["md", 85],
```
Comment (jonathan-fielding) [inline src/themes/money/theme.json:833]: looks like some magic numbers, are we not using standard theme spacing?

Comment (ruxiang05) [reply src/themes/money/theme.json:833]: @jonathan-fielding there's no number close enough to what design has given me

File: src/elements/link-list/src/index.tsx (hunk 2 above)
Comment (ruxiang05) [reply src/elements/link-list/src/index.tsx:28]: @jonathan-fielding it's not just the tile, it's also the icon

**Comment by ruxiang05** [comment]
Merged while GitHub was down so the PR was never closed but the commits are on master, closing this

**Pull request closed without merging**